translations/messages.da.yaml

```diff
@@ -4,7 +4,9 @@ frontpage:
 qr:
   title: 'Titel'
   description: 'Beskrivelse'
-  mode: 'Type'
+  mode:
+    title: 'Type'
+    help: '<b>Standard</b>: Qr-koden leder videre til destinationen.<br><b>Statisk sidevisning</b>: En dedikeret side vises med titlen og beskrivelsen.'
```

Comment:
Is it correct to include html in translations?

Reply:
After som research, it seems like the only way to go about this at the moment. I feel that it is necessary here to get the description of the modes that we want.